packages/paste-core/layout/media-object/__tests__/index.spec.tsx

```diff
@@ -1,143 +1,214 @@
+    expect(renderedMediaFigure).toHaveStyleRule('margin-left', '0.75rem');
   });
 });
 
 describe('MediaBody', () => {
   it('should render', (): void => {
-    const tree = renderer
-      .create(
-        <Theme.Provider theme="console">
-          <MediaBody>default</MediaBody>
-        </Theme.Provider>
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+    render(
+      <Theme.Provider theme="default">
+        <MediaBody data-testid="media-body">default</MediaBody>
+      </Theme.Provider>
+    );
+    const renderedMediaBody = screen.getByTestId('media-body');
+    expect(renderedMediaBody).toBeDefined();
   });
 
   it('should render as any HTML element', (): void => {
-    const wrapper: ReactWrapper = mount(<MediaBody as="article">default</MediaBody>);
-    expect(wrapper.exists('article')).toEqual(true);
-
-    const tree = renderer
-      .create(
-        <Theme.Provider theme="console">
-          <MediaBody as="article">default</MediaBody>
-        </Theme.Provider>
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+    render(
+      <Theme.Provider theme="default">
+        <MediaBody as="article" data-testid="media-body">
+          as article
+        </MediaBody>
+      </Theme.Provider>
+    );
+    const renderedMediaBody = screen.getByTestId('media-body');
+    expect(renderedMediaBody.nodeName).toEqual('ARTICLE');
+  });
+});
+
+describe('HTML attributes', () => {
+  it('should set a element data attribute for MediaObject', () => {
+    render(<MediaObject>media-object</MediaObject>);
+    expect(screen.getByText('media-object').getAttribute('data-paste-element')).toEqual('MEDIA_OBJECT');
+  });
+  it('should set a custom element data attribute for MediaObject', () => {
+    render(<MediaObject element="foo">media-object</MediaObject>);
+    expect(screen.getByText('media-object').getAttribute('data-paste-element')).toEqual('foo');
+  });
+  it('should set a element data attribute for MediaFigure', () => {
+    render(<MediaFigure>media-figure</MediaFigure>);
+    expect(screen.getByText('media-figure').getAttribute('data-paste-element')).toEqual('MEDIA_FIGURE');
+  });
+  it('should set a custom element data attribute for MediaFigure', () => {
+    render(<MediaFigure element="foo">media-figure</MediaFigure>);
+    expect(screen.getByText('media-figure').getAttribute('data-paste-element')).toEqual('foo');
+  });
+  it('should set a element data attribute for MediaBody', () => {
+    render(<MediaBody>media-body</MediaBody>);
+    expect(screen.getByText('media-body').getAttribute('data-paste-element')).toEqual('MEDIA_BODY');
+  });
+  it('should set a custom element data attribute for MediaBody', () => {
+    render(<MediaBody element="foo">media-body</MediaBody>);
+    expect(screen.getByText('media-body').getAttribute('data-paste-element')).toEqual('foo');
+  });
+});
+
+describe('Customization', () => {
+  it('should add custom styles to MediaObject', (): void => {
```

Comment:
Why is it that use `void` here? I think I asked this before and never looked back at what the answer was.

Reply:
All good. Linking to the other answer: https://github.com/twilio-labs/paste/pull/1641#discussion_r665720897